test/e2e/gateways/bgp-dynamic-neighbour/create_test.go

```diff
@@ -0,0 +1,89 @@
+package bgp_dynamic_neighbour
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"testing"
+
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
+	"github.com/spf13/cobra"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	"github.com/equinix/metal-cli/internal/gateway"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/equinix/metal-cli/test/helper"
+)
+
+func TestBgpDynamicNeighbours_Create(t *testing.T) {
```

Comment:
All of the new tests [are failing with 404s](https://github.com/equinix/metal-cli/actions/runs/7672035602/job/20912047460#step:5:114):

```
=== RUN   TestBgpDynamicNeighbours_Create/create_bgp_dynamic_neighbour
Error: Could not create BGP Dynamic Neighbour: 404 Not Found Not found
    helper.go:591: Could not create BGP Dynamic Neighbour: 404 Not Found Not found
--- FAIL: TestBgpDynamicNeighbours_Create (2.54s)
    --- FAIL: TestBgpDynamicNeighbours_Create/create_bgp_dynamic_neighbour (0.14s)
=== RUN   TestBgpDynamicNeighbours_Delete
    delete_test.go:27: Error when calling `VRFsApi.CreateBgpDynamicNeighbor`: 404 Not Found Not found
--- FAIL: TestBgpDynamicNeighbours_Delete (3.10s)
=== RUN   TestBgpDynamicNeighbours_Get
    get_test.go:25: Error when calling `VRFsApi.CreateBgpDynamicNeighbor`: 404 Not Found Not found
--- FAIL: TestBgpDynamicNeighbours_Get (2.65s)
=== RUN   TestBgpDynamicNeighbours_List
    list_test.go:25: Error when calling `VRFsApi.CreateBgpDynamicNeighbor`: 404 Not Found Not found
--- FAIL: TestBgpDynamicNeighbours_List (3.00s)
FAIL
```

Do we need to pass in any additional CLI flags?

Reply:
Not sure why its failing with 404 on the CI, its passing at local

```
~/go/src/metal-cli   gw-bgp-neighbours *2 !15 ?4 ❯ go test -v ./test/e2e/gateways/bgp-dynamic-neighbour/...                                                                          27s
=== RUN   TestBgpDynamicNeighbors_Create
=== RUN   TestBgpDynamicNeighbors_Create/create_bgp_dynamic_neighbor
--- PASS: TestBgpDynamicNeighbors_Create (6.15s)
    --- PASS: TestBgpDynamicNeighbors_Create/create_bgp_dynamic_neighbor (0.83s)
=== RUN   TestBgpDynamicNeighbors_Delete
=== RUN   TestBgpDynamicNeighbors_Delete/delete_bgpDynamicNeighbor_by_ID
--- PASS: TestBgpDynamicNeighbors_Delete (6.11s)
    --- PASS: TestBgpDynamicNeighbors_Delete/delete_bgpDynamicNeighbor_by_ID (0.82s)
=== RUN   TestBgpDynamicNeighbors_Get
=== RUN   TestBgpDynamicNeighbors_Get/get_bgpDynamicNeighbor_by_ID
--- PASS: TestBgpDynamicNeighbors_Get (5.97s)
    --- PASS: TestBgpDynamicNeighbors_Get/get_bgpDynamicNeighbor_by_ID (0.43s)
=== RUN   TestBgpDynamicNeighbors_List
=== RUN   TestBgpDynamicNeighbors_List/list_bgpDynamicNeighbor_by_ID
--- PASS: TestBgpDynamicNeighbors_List (6.04s)
    --- PASS: TestBgpDynamicNeighbors_List/list_bgpDynamicNeighbor_by_ID (0.66s)
PASS
ok  	github.com/equinix/metal-cli/test/e2e/gateways/bgp-dynamic-neighbour	24.716s
```

Reply:
I'm also getting the 404 Not found Not Found on my local tests.